types/index.d.ts

```diff
@@ -47,7 +47,7 @@ export interface DotenvConfigOutput {
 }
 
 /**
- * Loads `.env` file contents into {@link https://nodejs.org/api/process.html#process_process_env | `process.env`}.
```

Comment:
@ImRodry can you elaborate why the pipe `|` here causes issues with the documentation?

Reply:
I see you already merged this so you probably got the answer to this question but the issue was that the | was being put into the JSDoc comment and wasn’t just a separator in the `@link` tag.